### DISCUSSION TITLE
Is there any way to assign CC:T computers more power?

### DISCUSSION THREAD

**Discussion by YoshYoshTheBosh** [cc-tweaked/CC-Tweaked#2041] [Q&A]
I have to calculate an integral for a project of mine, but the calculation is so hard it often times out or takes a while, rendering it unusable with big numbers. is there any way to let CC:T use more power? or stop the "too long without yielding" error from occouring so soon after launch?

**Comment by zyxkad** [comment]
In `computercraft-server.toml`, under `[execution]` section, you can increase `max_main_global_time` and `max_main_computer_time`.
However the best way is check `os.epoch` while calculating and then manually yield by using `sleep(0)`

**Reply by SquidDev** [reply]
Those config options are used to configure rate limits for main-thread usage. They won't have any impact on the "Too long without yielding" error.
